Automattic/jetpack
# Rework the connection list to match the designs

Fixes https://github.com/Automattic/jetpack-reach/issues/330

## Proposed changes:

* Changed some strings to match the design
* Removed the table and added a list instead
* Used an expandable panel whee the user can mark the connection as shared or disconnect

### Other information:


## Jetpack product discussion


https://github.com/Automattic/jetpack-reach/issues/330
## Does this pull request change what data or activity we track or use?


No.
## Testing instructions:






[Figma file](https://www.figma.com/design/syO2iOAxaOb0DXVIfluarw/Social%3A-Connection-management-in-WPA?node-id=4428-3926&t=8FSmFNtnzwej5G5I-0)

* Go to the Social admin page
* Check that the connection list is looking alright and the same as on the Figma file
* The Add connection button should be secondary unless there are no connections
